src/components/tabs/tabs.component.js

```diff
@@ -114,7 +124,12 @@ const Tabs = ({
     (tabid) => {
       if (setLocation && !sidebarContext) {
         const url = `${_window.location.origin}${_window.location.pathname}#${tabid}`;
-        _window.history.replaceState(null, "change-tab", url);
+
+        if (isExternalHistory) {
```

Comment:
This doesn't look correct, the method shouldn't change based on `isExternalHistory`.

I think we need an additional prop that defaults to `replace`, the existing behaviour.
```
navigatorMethod: PropTypes.oneOf(['push', 'replace']);
```
then we would call
```js
historyAPI[navigatorMethod](url);
```